src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorEngineUnitTest.java

```diff
@@ -498,23 +398,21 @@ public void testAnnotationsHaveDescriptions() throws Exception {
 
         Set<String> sampleSet = Collections.singleton("FRED");
         final Set<VCFHeaderLine> headerInfo = new LinkedHashSet<>();
-        final List<String> annotationsToExclude= Collections.emptyList();
         final FeatureInput<VariantContext> dbSNPBinding = null;
         final List<FeatureInput<VariantContext>> features = Collections.emptyList();
 
-        headerInfo.addAll(VariantAnnotatorEngine.ofAllMinusExcluded(annotationsToExclude, dbSNPBinding, features, false).getVCFAnnotationDescriptions(false));
+        headerInfo.addAll(new VariantAnnotatorEngine(ofAllMinusExcluded(Collections.emptyList()), dbSNPBinding, features, false).getVCFAnnotationDescriptions(false));
 
         Assert.assertFalse(headerInfo.contains(null));
         new VCFHeader(headerInfo, sampleSet);//make sure this does not blow up: https://github.com/broadinstitute/gatk/issues/1713
     }
 
     @Test
     public void testNoNullInKeysAndDescriptions() throws Exception {
-        final List<String> annotationsToExclude= Collections.emptyList();
         final FeatureInput<VariantContext> dbSNPBinding = null;
         final List<FeatureInput<VariantContext>> features = Collections.emptyList();
 
-        final VariantAnnotatorEngine variantAnnotatorEngine = VariantAnnotatorEngine.ofAllMinusExcluded(annotationsToExclude, dbSNPBinding, features, false);
+        final VariantAnnotatorEngine variantAnnotatorEngine = new VariantAnnotatorEngine(ofAllMinusExcluded(Collections.emptyList()), dbSNPBinding, features, false);
```

Comment:
Why call `ofAllMinusExcluded` only to exclude nothing ?

Reply:
in order to get all annotations